projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.spec.ts

```diff
@@ -35,4 +48,10 @@ describe('OpenInAppComponent', () => {
 
     expect(currentLocation).toBe('mockRedirectUrl');
   });
+
+  it('should set the value `mobile_notification_expires_in` in session storage on dialog close', async () => {
+    component.onCloseDialog();
+    expect(sessionStorage.getItem('mobile_notification_expires_in')).not.toBe(null);
```

Comment:
instead of .not.toBe(null) you can use: toBeDefined()/toBeTruthy()/not.toBeNull()

Reply:
instead of .not.toBe(null) I have used now - not.toBeNull()